gnd/src/main/java/com/google/android/gnd/ui/home/featuredetails/FeatureDetailsFragment.java

```diff
@@ -89,12 +90,14 @@ public void onPrepareOptionsMenu(@NonNull Menu menu) {
             menuItem -> {
               if (menuItem.getItemId() == R.id.move_feature_menu_item) {
                 viewModel.getMoveMenuOptionVisible().observe(this, menuItem::setVisible);
+              } else if (menuItem.getItemId() == R.id.delete_feature_menu_item) {
```

Comment:
switch/case might be more appropriate here?

Reply:
Android Studio throws a warning with switch case.

```
Resource IDs will be non-final in Android Gradle Plugin version 7.0, avoid using them in switch case statements
```